gui.go

```diff
@@ -29,9 +29,18 @@ type Text struct {
 
 // SliderWannabe represents abstract of slider that changes a given parameter
 type SliderWannabe struct {
-	y         float64
-	parameter *Parameter
-	format    string
+	y           float64
+	canvasWidth float64
```

Comment:
Firstly I mistakenly thought , that`canvasWidth` is width of the rendering canvas. Maybe adding some more descriptive name would be suitable, also explanation in comment if needed.